libs/platform/src/lib/components/form/switch/switch/switch.config.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+import { ContentDensity } from '../../../form/form-control';
+
+/**
+ * Default options for platform Switch
+ */
+@Injectable({ providedIn: 'root' })
+export class SwitchConfig {
+    /**
+     * Content Density of element. 'cozy' | 'compact'
+     */
```

Comment:
We could have more generic config as content density can be applied to all the components like this. I remember Dima was showing me `PlatformConfig`

Reply:
It wasn't yet approved, so let's stick with component level configs, and later we could came up with platform/core/ or even all level configs and services

Reply:
sounds good if its one component but if we need to keep config for each new component like this, I would go in the next iteration to more generic approach and refactor this.

Reply:
components specific configs are must have anyway, over time they will have even more properties